Ladybird/FontPluginQt.cpp

```diff
@@ -31,6 +32,8 @@ FontPluginQt::FontPluginQt()
     Gfx::FontDatabase::set_default_font_query("Katica 10 400 0");
     Gfx::FontDatabase::set_fixed_width_font_query("Csilla 10 400 0");
 
+    Gfx::Emoji::set_emoji_lookup_path(String::from_deprecated_string(s_serenity_resource_root).release_value_but_fixme_should_propagate_errors());
```

Comment:
Seems rather confusing that this works differently compared to `set_default_fonts_lookup_path()` and takes a resource root, _not_ the actual lookup path

Reply:
The /res/emoiji part of the path is currently generated at build time. We can just generate the file name instead I guess.

Reply:
thanks for changing that :)